packages/ignite-cli/src/templates/plugin/boilerplate.js.ejs

```diff
@@ -0,0 +1,110 @@
+/**
```

Comment:
This is the most minimal boilerplate generator script I could come up with that still had enough functionality. We might want to refactor some of this out into boilerplate helper functions at some point.